src/index.ts

```diff
@@ -1 +1 @@
-export { verifyAuthorizationRequestCallback } from './verifyAuthorizationRequestCallback'
+export { verifyAuthorizationRequest } from './verifyAuthorizationRequestCallback'
```

Comment:
The file should probably also be renamed? And maybe we should add openid4vp into the name?